contribs/gmf/examples/editfeatureselector.html

```diff
@@ -28,9 +28,21 @@
         display: block;
         width: 30rem;
       }
+      gmf-editfeature {
+        display: block;
+      }
       .gmf-editfeatureselector-stopediting {
         float: right;
       }
+      .gmf-editfeature-attributes-container {
+        position: relative;
+      }
+      .gmf-editfeature-pending-panel {
+        background-color: rgba(230, 230, 230, 0.3);
+        height: 100%;
+        position: absolute;
+        width: 100%;
+      }
```

Comment:
It looks like you're adding a mask on top of the form container.
What about using a fieldset to disable the controls all at once?
http://getbootstrap.com/css/#forms-disabled-fieldsets